api/src/neo4j/queries/map.js

```diff
@@ -1,6 +1,38 @@
 import querySingleResult from 'neo4j/queryHandlers/single';
 import queryListResult from 'neo4j/queryHandlers/list';
 import parseParams from 'neo4j/shared/helper';
+const fs = require('fs');
+const readline = require('readline');
+
+const mapReactionIdSet = async (map) => {
+  const rl = readline.createInterface({
+      input: fs.createReadStream(`/project/svg/Human-GEM/${map.filename}`),
```

Comment:
Hard-coding `Human-GEM` here breaks when switching to the `Yeast-GEM` model, where the same functionality is supposed to work.